javascript/javascript1/week2/smart-ease.js

```diff
@@ -0,0 +1,180 @@
+// FLIGHT BOOKING FULLNAME FUNCTION
+
+function getFullname(firstname, surname, useFormalName, gender) {
+    if (firstname && surname && useFormalName == true && gender) {
```

Comment:
Prefer to use `===` then `==` since in your case would mean that would check string against string
```suggestion
    if (firstname && surname && useFormalName === true && gender) {
```